### DISCUSSION TITLE
[Bug?] Gh action run wont stop and fails to cancel

### DISCUSSION THREAD

**Discussion by fluoriteByte** [community/community#125011] [Actions]
### Select Topic Area

Bug

### Body

this is that action: https://github.com/fluoriteByte/org.virtualbox.VirtualBox-KVM.unofficial/actions/runs/9174375986
i have also sent a ticket to support but i just chose the category closest to gh actions since there was none

**Comment by FranciscoDoti** [comment]
I'm facing the same issue now. If I see the Github Status page, they say that everyhting is on green , operational, but maybe it's because no one has reported an incident yet. I'm facing the same issue now

**Reply by devmyipac** [reply]
Igualmente aqui, achei que era porque o meu workflow estava com algum erro mas estou achando que é um bug mesmo

**Reply by SolarLune** [reply]
I also am seeing this problem right now - jobs are stuck as queued, won't cancel, and since they won't end, build artifacts uploaded as the result of a job running are inaccessible as well. Interestingly, jobs that are listed as queued actually _seem_ to run, just not with the status updating.

**Comment by davevad93** [comment]
Hi @fluoriteByte ,

There is an incident occurring with `Actions` as indicated in [GitHub Status](https://www.githubstatus.com/incidents/nj47vccwm2zj), we can only wait for it to be fixed.

**Comment by grandmas-favorite** [comment]
Hi @fluoriteByte!

@davevad93 was correct, there was an incident this morning related to Actions that is tracked [here](https://www.githubstatus.com/incidents/nj47vccwm2zj), and the incident has now been resolved and Actions is operating normally!
